packages/calypso-e2e/src/lib/pages/signup/signup-domain-page.ts

```diff
@@ -20,9 +20,10 @@ export class SignupDomainPage {
 	 * Skips the domain selection screen.
 	 */
 	async skipDomainSelection(): Promise< void > {
-		await Promise.race( [
-			this.page.getByRole( 'button', { name: 'Skip Purchase', exact: true } ).click(),
-			this.page.getByRole( 'button', { name: 'Choose my domain later', exact: true } ).click(),
-		] );
+		console.log( 'async skipDomainSelection(): Promise< void > {' );
+		console.log( this.page.getByRole( 'button', { name: 'Choose my domain later', exact: true } ) );
+		console.log( this.page.getByRole( 'button', { name: 'Skip Purchase', exact: true } ) );
+		// await this.page.getByRole( 'button', { name: 'Skip Purchase', exact: true } ).click();
+		await this.page.getByRole( 'button', { name: 'Choose my domain later', exact: true } ).click();
```

Comment:
It looks like the only relevant line is the line 27. Let's make sure to clean this function up before deploying.